sasmodels/sasview_model.py

```diff
@@ -874,7 +874,8 @@ def test_cylinder():
     """
     Cylinder = _make_standard_model('cylinder')
     cylinder = Cylinder()
-    return cylinder.evalDistribution([0.1, 0.1])
+    # Smoke test: does it run without error?
+    cylinder.evalDistribution([0.1, 0.1])
```

Comment:
Do you not want to assert a kwown, expected returned value for `evalDistribution` here?